addons/base_vat/models/res_partner.py

```diff
@@ -76,6 +76,8 @@
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
+    vat = fields.Char(string="VAT/Tax ID")
```

Comment:
Why do you redefine the field here ?

Reply:
Hello @std-odoo 

it was changed based on the review of @LNAodoo 
field's label is changed in the XML from base_vat and in tracking, we are getting the original string in the tracking values.

Thanks

Reply:
Ok :)